modyn/supervisor/internal/triggers/drift/decision_policy.py

```diff
@@ -38,7 +41,16 @@ def evaluate_decision(self, distance: float) -> bool:
 
 class DynamicDecisionPolicy(DriftDecisionPolicy):
     """Decision policy that will make the binary is_drift decisions based on a
-    dynamic threshold."""
+    dynamic threshold.
+
+    We compare a new distance value with the series of previous distance values
+    and decide if it's more extreme than a certain percentile of the series. Therefore we count the
+    `num_more_extreme` values that are greater than the new distance and compare it with the
+    `percentile` threshold.
```

Comment:
For example here we could explain how we build this, or in the drift triggers.md

Reply:
I went for `DRIFT_TRIGGERS.md`

Reply:
I just copied a rough draft for the description of our solution; in my thesis, I'll use visualizations. It's rather hard to convey the issues here with plain text.